### DISCUSSION TITLE
Github not able to load all the class diagrams and other png files

### DISCUSSION THREAD

**Discussion by rohitkmrr** [tssovi/grokking-the-object-oriented-design-interview#19] [General]
Currently Github not able to load all the class diagrams and other png files in preview mode. 
**Solution:** 
Need to correct the path for all sources as 
replace "src="/" with "src=../" all over in https://github.com/tssovi/grokking-the-object-oriented-design-interview/tree/master/object-oriented-design-case-studies folder

**Comment by gituttammodi** [comment]
Kindly update the repo so images and diagrams are visible. Because it is not visible now.
